packages/headless/src/controllers/result-list/headless-result-list.ts

```diff
@@ -59,9 +60,12 @@ export function buildResultList(
       const state = engine.state;
 
       return {
-        results: state.search.response.results,
+        results: state.search.results,
         isLoading: state.search.isLoading,
       };
     },
+    fetchMoreResults() {
```

Comment:
This would benefit from a bit of jsdoc.

For inspiration: 

```
Using the same parameters as the last successful query, fetch another batch of results. Particularly useful for infinite scrolling, for example.
```